test-plan/eventing/channel/control-plane/channel-lifecycle-conformance.md

```diff
@@ -0,0 +1,228 @@
+# Channel Lifecycle 
```

Comment:
This is a comment about where this file is located. I don't know the original intent but I would expect `channel-lifecycle-conformance.md` to be under `test-plan/eventing/` and the yaml files under ` test-plan/eventing/control-plane/channel-lifecycle/`

/cc @salaboy

Reply:
Seems a little bit more organized to me this way but you decide :)